### PR TITLE
RavenDB-11773 Forbid specifying the same port that server should be u…

### PR DESCRIPTION
**Pull request by ml054**
…sing as current setup wizard is

### CODE REVIEW

File: src/Raven.Studio/typescript/models/wizard/unsecureSetup.ts [hunk 1]
```diff
@@ -28,12 +28,28 @@ class unsecureSetup {
     }
     
     private initValidation() {
+        const currentHttpPort = window.location.port || "80";
```
Comment (ayende) [inline src/Raven.Studio/typescript/models/wizard/unsecureSetup.ts:31]: Technically, they can run the setup while using HTTPS if they re-run by editing the `settings.json` to say `SetupMode=Initial`. Can we add the check for 443 here?

Comment (ppekrol) [reply src/Raven.Studio/typescript/models/wizard/unsecureSetup.ts:31]: Yes

Comment (ppekrol) [reply src/Raven.Studio/typescript/models/wizard/unsecureSetup.ts:31]:
```
const currentHttpPort = window.location.port || (window.location.protocol === "https:" ? "443" : "80")
```

Comment (ayende) [reply src/Raven.Studio/typescript/models/wizard/unsecureSetup.ts:31]: Yes

**Pull request merged**